package.json

```diff
@@ -8,12 +8,8 @@
     "npm": ">=9.5.0"
   },
   "scripts": {
-    "build": "tsc -p tsconfig.json && tsc -p tsconfig-cjs.json",
-    "postinstall": "husky install",
-    "prepublishOnly": "npm run build",
```

Comment:
Why are we removing these and `prepare` script please? These stem from the template-typescript-package package

Reply:
bump, I think we should keep `postinstall`